af/oometa/006-test-lists.rollback.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+
+select _v.unregister_patch( '006-test-lists' );
+
+DROP SEQUENCE cat_no_seq;
+
+DROP TABLE url_category;
+
+DROP SEQUENCE IF EXISTS url_no_seq;
+
+DROP TABLE IF EXISTS url;
+
+DROP SEQUENCE IF EXISTS country_no_seq;
+
+DROP TABLE IF NOT EXISTS country;
```

Comment:
Is there any reason to preserve this IFs?

Reply:
It's slightly more convenient to have them for development and you want to do a partial rollback.

Reply:
You should feel free to drop them when you deploy it.